influx/authorization.go

```diff
@@ -70,7 +71,10 @@ func (b *BearerJWT) Set(r *http.Request) error {
 
 // Token returns the expected InfluxDB JWT signed with the sharedSecret
 func (b *BearerJWT) Token(username string) (string, error) {
```

Comment:
Just curious if Chronograf have access to the same shared secret as InfluxDB and Kapacitor?

Reply:
Hmm, somehow this was buried in my email and I didn't respond. sorry @desa!  Yes, chronograf can be configured to have the same shared secret.  That way it can auth against meta nodes and such.